xsd/NeTEx_publication.xsd

```diff
@@ -7241,7 +7431,7 @@ Provides a general purpose wrapper for NeTEx data content.</xsd:documentation>
 			<xsd:annotation>
 				<xsd:documentation>Every [FareTable Id + Version] must be unique within document.</xsd:documentation>
 			</xsd:annotation>
-			<xsd:selector xpath=".//netex:FareTable"/>
+			<xsd:selector xpath=".//netex:FareTable   | .//netex:StandardFareTable"/>
```

Comment:
Many of the Fare selectors are using multiple spaces around |

Reply:
Have also corrected these where spottet, but probably heaps more in the code. Though cosmetic, does not affect the functionality, these and similar issues should be cleaned up iterative when spotted.